Controls/CkEditorMinimal/src/DotVVM.Contrib/CkEditorMinimal.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using DotVVM.Framework.Controls;
+using DotVVM.Framework.Hosting;
+using DotVVM.Framework.Binding;
+
+namespace DotVVM.Contrib
+{
+    /// <summary>
+    /// Renders a ...
+    /// </summary>
+    public class CkEditorMinimal : HtmlGenericControl
+    {
+
+        public CkEditorMinimal(): base("textarea") 
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the text in the control.
+        /// </summary>
+        [MarkupOptions(Required = true)]
+        public string Html
+        {
+            get { return Convert.ToString(GetValue(HtmlProperty)); }
+            set { SetValue(HtmlProperty, value); }
+        }
+
+        public static readonly DotvvmProperty HtmlProperty =
+            DotvvmProperty.Register<string, CkEditorMinimal>(t => t.Html, "");
+
+
+        protected override void OnPreRender(IDotvvmRequestContext context)
+        {
+            context.ResourceManager.AddRequiredResource("dotvvm.contrib.CkEditorMinimal");
+            context.ResourceManager.AddRequiredResource("ckeditor-config");
+            base.OnPreRender(context);
+        }
+
+        protected override void AddAttributesToRender(IHtmlWriter writer, IDotvvmRequestContext context)
+        {
+            base.AddAttributesToRender(writer, context);
```

Comment:
Because the binding handler requires the element to have `id` attribute, I think it would be worth
* checking if the ID is not null if you want to remain explicit
* or generating `data-dotvvm-id` attribute using `this.AddDotvvmUniqueIdAttribute()` to avoid annoying the user with the ID
* or checking if the ID is null and generating the unique id if it is, for the combination of both (this is my favorite option ;) )